crates/ruff_linter/src/rules/ruff/rules/unnecessary_cast_to_int.rs

```diff
@@ -117,65 +107,6 @@ fn single_argument_to_int_call<'a>(
     Some(argument)
 }
 
-/// Returns an [`Edit`] when the call is of any of the forms:
-/// * `round(integer)`, `round(integer, 0)`, `round(integer, None)`
-/// * `round(whatever)`, `round(whatever, None)`
-fn replace_with_shortened_round_call(
```

Comment:
I think this now removes too much. We still need to validate that the `round` call returns a number, that is, that it has an `number` argument and the `ndigits` argument is either absent, `None`, or `0`.